grpc/api.proto

```diff
@@ -0,0 +1,38 @@
+syntax = "proto3";
+
+option go_package = "github.com/awfm9/flow-dps/grpc";
+
+service API {
+  rpc GetRegister (GetRegisterRequest) returns (Register) {}
+  rpc GetValues (GetValuesRequest) returns (Values) {}
```

Comment:
I find the naming a bit strange. You make the request naming super long, and then the response super short.

I personally would use the full explicit name for both for the sake of consistenc (`GetRegisterResponse` and `GetValuesResponse`).